lib/todo_v1/ricky_main.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: const MyHomePage(title: 'To Do App'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  const MyHomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  State<MyHomePage> createState() => _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  bool checkedValue = false;
```

Comment:
this checkedValue is shared across all checkboxes? Consider making it a list so that each checkbox has its own value

Reply:
A better way to do this is to change the List<String> list to a List<Todo>, where Todo will be a custom class with own properties. 

```
class Todo {

  // Add your property here, e.g. name and isChecked 
}
```